internal/checker/checker.go

```diff
@@ -183,6 +182,19 @@ func (c *Checker) LoadIndex(ctx context.Context) (hints []error, errs []error) {
 		errs = append(errs, err)
 	}
 
+	c.masterIndex.MergeFinalIndexes()
+
+	for blob := range c.masterIndex.Each(ctx) {
+		// compute pack size using index entries
+		size, ok := c.packs[blob.PackID]
+		blobsize := int64(pack.PackedSizeOfBlob(blob.Length))
+		if ok {
```

Comment:
I'd change this to
```
if !ok {
    size = pack.HeaderSize
}
c.packs[blob.PackID] = size + blobsize
```

Reply:
changed (and directly used the definition of `blobsize` as it is now only used here)